kubectl-plugin/pkg/cmd/create/create_cluster.go

```diff
@@ -77,6 +79,8 @@ func NewCreateClusterCommand(streams genericclioptions.IOStreams) *cobra.Command
 
 	cmd.Flags().StringVar(&options.rayVersion, "ray-version", util.RayVersion, "Ray version to use")
 	cmd.Flags().StringVar(&options.image, "image", fmt.Sprintf("rayproject/ray:%s", options.rayVersion), "container image to use")
+	cmd.Flags().StringToStringVar(&options.labels, "labels", options.labels, "K8s labels, e.g. '--labels lazuli=bunting,blue=grosbeak'")
```

Comment:
Add examples to `createClusterExample` for these 2 new flags as well?

Reply:
sgtm, updated